cosmwasm/ibc-union/lightclient/ethereum/src/client.rs

```diff
@@ -597,10 +597,10 @@ mod tests {
             slot: INITIAL_HEADER.slot,
             state_root: INITIAL_HEADER.state_root,
             storage_root: INITIAL_STORAGE_HASH,
-            timestamp: compute_timestamp_at_slot::<Mainnet>(
+            timestamp: Timestamp::from_secs(compute_timestamp_at_slot::<Mainnet>(
                 SEPOLIA_GENESIS_TIME,
                 INITIAL_HEADER.slot,
-            ),
+            )),
```

Comment:
@aeryz can you confirm this is the intended behaviour?

Reply:
Onayliyorum

Reply:
discussed internally and realized that while the tests were "incorrect" before (storing the timestamp as seconds instead of nanoseconds), it didn't cause any issues since the timestamp is only read when doing packet verification.